submissions/migrations/0003_submission_null_FK.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='student_item',
+            field=models.ForeignKey(to='submissions.StudentItem', null=True),
+        ),
+    ]
```

Comment:
One potential issue I've discovered during dev testing - trying to reverse this migration after NULL values have been added results in an error, so I'll have to figure that out, at least.

`django.db.utils.OperationalError: (1138, 'Invalid use of NULL value')`.